### ISSUE TITLE
add #[derive(Clone)] on models 

### ISSUE THREAD

**Issue by VRichardJP** [MingweiSamuel/Riven#31]
I see that none of the structs defined in `src/models.rs` implements the `Clone` trait, which would be very useful to extract data.

For example let's say I want to extract ban information from a game, I cannot do something like:
```rust
let bans: Vec<Ban> = match_data.info.teams.iter().map(|t| t.bans.clone()).flatten().collect();
```

Yet `Ban` is a struct only containing `Champion` and `i32`, which are both `Clone` (and even `Copy`), so it could simply derive `Clone`. It seems to me that other structures in `src/models.rs` could also derive the `Clone` trait. Would it be possible to make this change?

**Comment by MingweiSamuel** [comment]
Good catch

**Comment by MingweiSamuel** [comment]
Fixed in 0e22623056f439b0af40522eb0ff82debf5731b4  
Released in https://crates.io/crates/riven/2.2.0

(Not sure about the other derives (`Ord`, `Eq`, `Hash`), maybe those will be useful too but the semantics might not fully match what people expect (i.e. for `Summoner`))
